test cases/failing/93 vala without c/meson.build

```diff
@@ -1,2 +0,0 @@
-project('vala without c')
-add_languages('vala')
```

Comment:
This should be moved to non-failing tests instead of deleting.

Reply:
Let's just remove c from an existing test, to save on test time. I'll send a patch in s bit